optimize/self-managed/optimize-deployment/version-policy.md

```diff
@@ -10,24 +10,13 @@ Camunda Optimize versions are denoted as X.Y.Z as well as by an optional [pre-re
 
 ## Release Cadence
 
-Camunda Optimize will get new releases every month being at least an alpha release and/or a patch release. At the end of a quarter either a new minor/major or a preview release will be published. The following table illustrates when to expect which type of release.
-
-| Quarter | Type of Release |
-| ------- | --------------- |
-| Q1      | Minor/Major     |
-| Q2      | Preview         |
-| Q3      | Minor/Major     |
-| Q4      | Preview         |
+Camunda Optimize will get new releases every month being at least an alpha release and/or a patch release. Every six months, a new minor/major will be published. The release dates are be aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).
```

Comment:
```suggestion
Camunda Optimize will get a new release every month. Every six months, a new minor/major will be published, all other months it will be an alpha and/or patch release. The release dates are aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).
```
Or sth else for the first sentence, phrasing just felt off